contrib/opencensus-ext-azure/tests/test_azure_standard_metrics.py

```diff
@@ -12,51 +12,263 @@
+
+            self.assertEqual(mem, 100)
+
+    @mock.patch('opencensus.ext.azure.metrics_exporter'
+                '.standard_metrics.process.logger')
+    def test_get_process_private_bytes_exception(self, logger_mock):
+        with mock.patch('opencensus.ext.azure.metrics_exporter' +
+                        '.standard_metrics.process.PROCESS') as process_mock:
+            process_mock.memory_info.side_effect = Exception()
+            standard_metrics.ProcessMemoryMetric.get_value()
+
+            logger_mock.exception.assert_called()
+
+    def test_get_processor_time_metric(self):
+        metric = standard_metrics.ProcessorTimeMetric()
+        gauge = metric()
+
+        self.assertEqual(gauge.descriptor.name,
+                         '\\Processor(_Total)\\% Processor Time')
+
+    def test_get_processor_time(self):
+        with mock.patch('psutil.cpu_times_percent') as processor_mock:
+            cpu = collections.namedtuple('cpu', 'idle')
+            cpu_times = cpu(idle=94.5)
+            processor_mock.return_value = cpu_times
+            processor_time = standard_metrics.ProcessorTimeMetric.get_value()
+
+            self.assertEqual(processor_time, 5.5)
+
+    def test_get_process_cpu_usage_metric(self):
+        metric = standard_metrics.ProcessCPUMetric()
+        gauge = metric()
+
+        self.assertEqual(gauge.descriptor.name,
+                         '\\Process(??APP_WIN32_PROC??)\\% Processor Time')
+
+    @mock.patch('opencensus.ext.azure.metrics_exporter'
+                '.standard_metrics.process.psutil')
+    def test_get_process_cpu_usage(self, psutil_mock):
+        with mock.patch('opencensus.ext.azure.metrics_exporter'
+                        '.standard_metrics.process.PROCESS') as process_mock:
+            process_mock.cpu_percent.return_value = 44.4
+            psutil_mock.cpu_count.return_value = 2
+            cpu_usage = standard_metrics.ProcessCPUMetric.get_value()
+
+            self.assertEqual(cpu_usage, 22.2)
+
+    @mock.patch('opencensus.ext.azure.metrics_exporter'
+                '.standard_metrics.process.logger')
+    def test_get_process_cpu_usage_exception(self, logger_mock):
+        with mock.patch('opencensus.ext.azure.metrics_exporter'
+                        '.standard_metrics.process.psutil') as psutil_mock:
+            psutil_mock.cpu_count.return_value = None
+            standard_metrics.ProcessCPUMetric.get_value()
+
+            logger_mock.exception.assert_called()
+
+    def test_request_patch(self):
+        map = standard_metrics.http_requests.requests_map
+        func = mock.Mock()
+        new_func = standard_metrics.http_requests.request_patch(func)
+        new_func()
+
+        self.assertEqual(map['count'], 1)
+        self.assertIsNotNone(map['duration'])
+        self.assertEqual(len(func.call_args_list), 1)
+
+    def test_server_patch(self):
+        standard_metrics. \
+            http_requests. \
+            ORIGINAL_CONSTRUCTOR = lambda x, y, z: None
```

Comment:
Suggestion on formatting, but it might be too wide?
```suggestion
        standard_metrics.http_requests.ORIGINAL_CONSTRUCTOR = (
            lambda x, y, z: None
        )
```